src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -138,8 +139,8 @@ const removeFailedConnectionsFromDb = async (logger: Logger, installations: Inst
 					host: jiraHost,
 					gitHubAppId
 				});
-			} catch (err) {
-				const deleteSubscriptionError = `Failed to delete subscription: ${err}`;
+			} catch (err: unknown) {
+				const deleteSubscriptionError = `Failed to delete subscription: ${errorStringFromUnknown(err)}`;
 				logger.error(deleteSubscriptionError);
```

Comment:
since you are touching this, would you mind also to change this line to 

```
logger.error({ err }, deleteSubscriptionError);
```
🙏  We usually pass { err } to logger in case of an exception